docs/multiregion.md

```diff
@@ -0,0 +1,98 @@
+* installation is handled via the superCluster operator
+
+Cons:
+* the latency issue still remains. (e.g. request for creating new service instance would propagate from user -- superClusterManager -- region -- region local serviceCluster)
+* the superCluster becomes Single point of Failure
+* the superCluster has admin privileges in the region's management cluster. A exploit in our API server implementation could lead to unwanted privilege escalation.
+
+## Proposed solution:
+
+### Having no superCluster/Federations
+
+* The latency and SPOF issue shall still reamain.
+* We build API server accepting user's kubeconfig/token
+    * This way we're preventing privilege escalation since API server doesen't have cluster-admin privileges.
+* it's simpler approach
+
+### build truly decentralized KubeCarrier installation with helper API/library for cross-region management
+
+The way cross-region requires consists of two operation types:
+* routing
+* fan-out
+
+Also there are two ways for keeping configuration in sync:
+* static configuration
+* central configuration plane (akin to superCluster in the original @nico proposal)
+
+The routing operations require the client be routed to the right region executing its request. (e.g. create new serviceCluster, tenant consuming new service, etc.). The typical workflow would work like this:
+
+* client looking up the cache does he have a route in place
+    * if not ask the central configuration plane
+    * alternatively fail since the region are hard-coded for the client
+* handle the authentication. The authorization is already handled by the account, and general KubeCarrier installation. The authentication could work in multiple ways:
+    * each kubecarrier region management cluster having the same credentials (e.g. same OIDC settings, or same client CA, or whatever)
+* perform the required operation
+
+The fan-out operations are more complicated, but rarer. They include the following:
+* adding/removing regions
+* adding/removing accounts
+* adding/removing serviceCluster (if we want pair account-serviceCluster globally unique)
+
+Their application differ depending on the central configuration store presence. In case the central configuration store is present, add the fan-out operation to the write-ahead log, perform it client side, and report the result. Upon next fan-out operation check the write-ahead log for any possible failure and required clean ups.
+
+In case no central configuration store is present, due to limited operation types that can be done using the known distributed system techniques, it's also a non-issue. For service cluster, for example could work like this:
+
+* append this operation to write-ahead log
+* create ServiceCluster reservation in every cluster
+* convert the target ServiceCluster from reservation to the configuration
```

Comment:
Is this like a leader-election?
And what does the `write-ahead` log mean here? Does it present on every KubeCarrier installations, or only on the one with central configuration?

Reply:
leader-election is different. `write-ahead` log means that you're writing your intentions somewhere, probably in every KubeCarrier installation. It's a bit WIP